Print_A_Bot/controls/models.py

```diff
@@ -11,13 +13,18 @@ class LightShow(models.Model):
     def __str__(self):
         return self.name
 
+
 class LightShowStep(models.Model):
     show = models.ForeignKey(LightShow)
     light = models.IntegerField(choices=settings.LED_CHOICES)
-    red = models.IntegerField()
-    green = models.IntegerField()
-    blue = models.IntegerField()
-    order = models.IntegerField()
+    color = RGBColorField(default="green")
+    red = models.IntegerField(blank=True, null=True)
+    green = models.IntegerField(blank=True, null=True)
+    blue = models.IntegerField(blank=True, null=True)
+    order = models.IntegerField(blank=True, null=True)
+
+    def hex(self):
```

Comment:
this reads like it should be a `@property`

possibly with a `#` prepended to the string